pkg/services/baremetal/host/host.go

```diff
@@ -1160,10 +1196,14 @@ func (s *Service) actionImageInstalling() actionResult {
 	// Execute install image
 	out = sshClient.ExecuteInstallImage(postInstallScript != "")
 	if out.Err != nil {
-		record.Warnf(s.scope.HetznerBareMetalHost, "ExecuteInstallImageFailed", out.StdOut)
+		record.Warnf(s.scope.HetznerBareMetalHost, "ExecuteInstallImageFailed", fmt.Sprintf("Error: %s. %s", out.Err.Error(), out.StdOut))
```

Comment:
just asking: are you sure that there is nothing in StdErr?

Reply:
@janiskemper good point. Pretty printing out.Stdout, out.Stderr, out.Err is "hard". I created a method: Output.String(). Please have a look: https://github.com/syself/cluster-api-provider-hetzner/pull/1407/commits/0c86f783f905d82667159029923fcbc997093bfd

If I like it, then we can use that method in several places. But maybe in a second PR.

Reply:
yes I like it!